### DISCUSSION TITLE
Climate change impacts and APSIMX

### DISCUSSION THREAD

**Discussion by LenLon** [APSIMInitiative/ApsimX#8485] [Ideas]
I study climate change impacts on cereal crops, both longterm using climate model projections and current using counterfactual climate model data. APSIMX should be highly suitable for simulating climate change impacts on e.g. crop yields and the mediating impact of different adaptation strategies, but in my perusal of the issues in the github and the available documentation I have found two major shortcomings:

- APSIMX has no parametrisation of crop stresses relating (high) temperature whatsoever (I would love to be wrong on this)

- APSIMX' CO2 response function does not work for low CO2 values < 350 ppm

Temperature increase is almost certain under all climate change scenarios, with temperature regimes previously not experienced on earth in the late 21st century, especially in the already hot tropics. While I understand that crop responses to extreme temperatures are not well-researched, APSIMclassic already had some implementation of heat stress (at least in the case for grain formation in maize) which became the main limiting factor in yield formation for temperature increases of +3 deg C in the subtropics in my studies, much more so than lack of water. Water availability or rainfall is additionally much more difficult to project than temperature irregardless of climate change scenario. So why is any implementation of this currently missing in APSIMX? A lack of time, or too much uncertainty in the implemented response? In my opinion, this neglects a major yield-limiting factor of climate change. This is not to criticise the APSIMX team, but rather to stimulate some discussion around the how's and why's of this underappreciated topic. 

The CO2 response function's limitation to values above 350 ppm is a hard limit on APSIMX's applicability for studies attempting to attribute climate change impacts on agriculture already happening now. While CO2 fertilisation may be of limited importance for C4 crops such as maize, the fact that APSIMX throws a fatal error for values < 350 ppm precludes its inclusion in attribution studies, where counterfactual climates (i.e. climates without anthropogenic forcing, so pre-industrial CO2 conc. in the atmosphere) are used as a baseline to which one can compare current climates and yields. I do not know if this is an easy fix, but as climate change attribution is a growing discipline that generates a lot of interesting results I see a lot of potential of including process-based models to facilitate impact attribution. 

If anyone has any pointers (I am not shy about trying other crop models that can do this, APSIMclassic maybe?) on how to get around these limitations or how one would deal with similar issues in climate change studies, I'd be greatly appreciative.

Looking forward to hearing from the community!

**Comment by hol353** [comment]
Tagging some teams to this discussion as the new discussion forum on GitHub may be missed by lots of people.

@APSIMInitiative/reference-panel
@APSIMInitiative/developers
